torch_xla/csrc/ops/count_nonzero.cpp

```diff
@@ -0,0 +1,50 @@
+#include "torch_xla/csrc/ops/count_nonzero.h"
+
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/xla_lower_util.h"
+
+namespace torch_xla {
+namespace {
+
+xla::Shape NodeOutputShape(const torch::lazy::Value& input,
+                           std::vector<int64_t> dims) {
+  xla::Shape input_shape = GetXlaShape(input);
+  std::vector<int64_t> dimensions;
+  if (dims.empty()) {
```

Comment:
you can do 
`if (!dims.empty())` for line 17-> 22, and remove line 14. since return logic is the same.

Reply:
done